lib/elixir/lib/gen_server.ex

```diff
@@ -982,11 +982,13 @@ defmodule GenServer do
 
   `c:handle_cast/2` will be called on the server to handle
   the request. In case the `server` is on a node which is
-  not yet connected to the caller one, the call is going to
-  block until a connection happens. This is different than
-  the behaviour in OTP's `:gen_server` where the message
-  is sent by another process in this case, which could cause
-  messages to other nodes to arrive out of order.
+  not yet connected to the caller one, the semantics differ
+  depending on the used Erlang/OTP version.
+
+  Before Erlang/OTP 21, the call is going to block until a
+  connection happens. This was done to guarantee ordering
```

Comment:
I think this is not accurate, because the changes were made to guarantee ordering in Elixir’s one, not Erlang’s.

One thing that is not clear is what happens with the ordering in newer Elixir/Erlang versions.

Reply:
> I think this is not accurate, because the changes were made to guarantee ordering in Elixir’s one, not Erlang’s.

You are right, adjusted in d8d5a1a97d0ce7c8ffaa4d21a3731433e88efbd9.